frontend/src/text_resources.js

```diff
@@ -492,7 +492,12 @@ let LOCALIZE = new LocalizedStrings({
       passwordConfirmationRequirements: "It must match your password",
       dobDayField: "Day field selected",
       dobMonthField: "Month field selected",
-      dobYearField: "Year field selected"
+      dobYearField: "Year field selected",
+      emailsList: "emails list",
```

Comment:
Should these start with capital letters?

Reply:
Updated with capital letters.
See commit #https://github.com/code-for-canada/project-thundercat/pull/476/commits/6393e48cf058183318318a7a71f940e18b9d9812